smalltalksrc/Slang/CSlangPrettyPrinter.class.st

```diff
@@ -16,6 +16,58 @@ CSlangPrettyPrinter >> contents [
 	^ stream contents
 ]
 
+{ #category : 'testing' }
+CSlangPrettyPrinter >> expressionListHasNonCommentStatementAfter: expressions futureIndex: futureIndex [
+	"we found a begin inlining comment in an expressionList, sometimes the last statements of an expression list are serie of comment, we iterate the next element to see if there is an actual statement after otherwise the next part is not an element of the list"
```

Comment:
```suggestion
	"Find a begin inlining comment in an expressionList starting at futureIndex.
	Sometimes the last statements of an expression list are a sequence of comments.
	We iterate the next elements to see if there is an actual statement after otherwise the next part is not an element of the list"
```